vm/src/stdlib/os.rs

```diff
@@ -1127,6 +1127,95 @@ pub(super) mod _os {
         OutputMode::String.process_path(curdir_inner(vm)?, vm)
     }
 
+    #[cfg(unix)]
+    #[pyfunction]
+    fn register_at_fork(kwargs: crate::function::KwArgs, vm: &VirtualMachine) -> PyResult<()> {
+        let mut match_found = false; // better way to handle this?
+        for (key, value) in kwargs.into_iter() {
+            if !value.is_callable() {
+                return Err(vm.new_type_error("Args must be callable".to_owned()));
+            }
+            if key == "before" {
```

Comment:
rust conventions prefer `match key {`

Reply:
Can this be rewritten using a struct with `#[derive(FromArgs)]`?

Reply:
Rewriting using `#[derive(FromArgs)]`